pkg/virt-api/webhooks/validating-webhook/validating-webhook_test.go

```diff
@@ -58,6 +58,10 @@ var _ = Describe("Validating Webhook", func() {
 		})
 	})
 
+	AfterEach(func() {
+		os.Setenv("FEATURE_GATES", "")
```

Comment:
This can have unexpected side effects over time. It would be better to save the "FEATURE_GATES" varaible from before the test and restore that.